RedHatInsights/sed-frontend
# feat(RHINENG-7052): Update to FEC

# Description

Resolves: https://issues.redhat.com/browse/RHINENG-7052



# How to test the PR

1. Run the application
2. Make sure there are no changes and the app functions as expected

# Before the change


# After the change


# Dependent work link


# Checklist:
